eth/beacon/state_machines/configs.py

```diff
@@ -0,0 +1,18 @@
+from typing import (
+    NamedTuple,
+)
+
+BeaconConfig = NamedTuple(
+    'BeaconConfig',
+    (
+        ('BASE_REWARD_QUOTIENT', int),
+        ('DEFAULT_END_DYNASTY', int),
+        ('DEPOSIT_SIZE', int),
+        ('CYCLE_LENGTH', int),
+        ('MIN_COMMITTEE_SIZE', int),
+        ('MIN_DYNASTY_LENGTH', int),
+        ('SHARD_COUNT', int),
+        ('SLOT_DURATION', int),
```

Comment:
As noted in another comment, the following two are good candidates to use a `Second` `NewType` rather than using an `int`.